### ISSUE TITLE
Error in generating thumbnails

### ISSUE THREAD

**Issue by HeDo88TH** [DroneDB/DroneDB#77]
Trying to generate the thumbnails of one image gave me this error:

```
ERROR 6: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
ERROR 1: C:/Users/IT 2/Pictures/DJI/test/DJI_0347.jpg, band 1: IReadBlock failed at X offset 0, Y offset 0: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
ERROR 6: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
ERROR 1: C:/Users/IT 2/Pictures/DJI/test/DJI_0347.jpg, band 2: IReadBlock failed at X offset 0, Y offset 0: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
ERROR 6: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
ERROR 1: C:/Users/IT 2/Pictures/DJI/test/DJI_0347.jpg, band 3: IReadBlock failed at X offset 0, Y offset 0: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
ERROR 6: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
ERROR 1: C:/Users/IT 2/Pictures/DJI/test/DJI_0347.jpg, band 1: IReadBlock failed at X offset 0, Y offset 0: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
ERROR 1: IReadBlock failed at X offset 0, Y offset 0: C:/Users/IT 2/Pictures/DJI/test/DJI_0347.jpg, band 1: IReadBlock failed at X offset 0, Y offset 0: Reading this image would require libjpeg to allocate at least 107811081 bytes. This is disabled since above the 104857600 threshold. You may override this restriction by defining the GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC environment variable, or recompile GDAL by defining the GDAL_LIBJPEG_LARGEST_MEM_ALLOC macro to a value greater than 104857600
output/DJI_0347.jpg

C:\Users\IT 2\Pictures\DJI\test>ddb info DJI_0347.jpg
Path: file://C:/Users/IT 2/Pictures/DJI/test/DJI_0347.jpg
Type: GeoImage (3)
CameraPitch: -28.200000762939453
CameraRoll: 0.0
CameraYaw: 147.10000610351563
CaptureTime: 1600113013000.0
Height: 2997
Make: DJI
Model: FC3170
Orientation: 1
Sensor: dji fc3170
Width: 3997
Modified Time: 1600116238
Size: 2.77 MB
Point Geometry: [[9.046611111111, 45.49890833333, 67.5] ]
```
This is the picture
![DJI_0347](https://user-images.githubusercontent.com/7868983/93144470-5958e680-f6ea-11ea-908a-80c709b29695.jpg)

I think this could be fixed by defining GDAL_ALLOW_LARGE_LIBJPEG_MEM_ALLOC

**Comment by pierotofy** [comment]
Related: https://github.com/OSGeo/gdal/pull/2439

**Comment by pierotofy** [comment]
Should be fixed with #78
